ios/FluentUI/Vnext/Persona/PersonaView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+@objc public protocol PersonaCellProtocol {
+    @objc var persona: MSFAvatarState? { get set }
+    @objc var titleTrailingAccessoryView: UIView? { get set }
+    @objc var subtitleTrailingAccessoryView: UIView? { get set }
+    @objc var onTapAction: (() -> Void)? { get set }
+}
+
+@objc public class MSFPersonaCellState: MSFListCellState, PersonaCellProtocol {
+    @objc @Published public var persona: MSFAvatarState?
+}
+
+struct PersonaView: View {
+    @ObservedObject var state: MSFPersonaCellState
+    @ObservedObject var tokens: MSFListCellTokens
+
+    init(state: MSFPersonaCellState, windowProvider: FluentUIWindowProvider?) {
```

Comment:
We can create the MSFPersonaCellState internally and just expose it instead of getting a init param here.